### DISCUSSION TITLE
Point of improvements

### DISCUSSION THREAD

**Discussion by itsManjeet** [itsManjeet/avyos#25] [General]
Based on the reviews and comments on [Distrowatch](https://distrowatch.com/weekly.php?issue=20230529#rlxos)  by Jesse Smith

## Installation Media

- [x] Boot legacy support.
- [x] Can't change System Installer Language (Only English)?
- [x] Installer is not waiting on the logs window after failure.
- [x] Gnome Shell power menu is not visible.

## Operating System
- [x] No information of updates (size, changelog) and no progress.
- [x] No manual entry for <everything> (need to include **docs** cache space)

## Package Management
- [x] Using Layers for provide extra packages support that can't be distributed using flatpaks.

## Extra
- [ ] The bolt greeting module sucks... (always)
- [x] Documentations...

**Comment by ghost** [comment]
Aren't things too easy to be documented 😅?

**Reply by itsManjeet** [reply]
😅 not for newcomers

**Reply by itsManjeet** [reply]
Docs finish krte h is week phele,

**Comment by ZachjuKamashi** [comment]
When might Boot legacy support be added? I'm wanting to try it out, but I don't really have UEFI devices to install it to.

**Reply by ghost** [reply]
Experimental release will be available after the mid of October with BIOS boot support
The stable channel and iso is planned on the mid of November

You can also follow the project board for progress
https://github.com/users/itsManjeet/projects/10
